src/ontodia/diagram/view.ts

```diff
@@ -432,9 +432,122 @@ export class DiagramView extends Backbone.Model {
                 end: this.createLinkMarker(linkTypeId, false, style.markerTarget),
             };
         }
+        style.router = this.getLinkRouter();
         return style;
     }
 
+    private getLinkRouter(): LinkRouter {
+        return (vertices: Vertex[], args: RouterProps, linkView: LinkView): Vertex[] => {
+            const link = linkView.model;
+            if (!vertices) {
+                vertices = [];
+            }
+            // If the cell is a view, find its model.
+
+            // The cell is a link. Let's find its source and target models.
+            let srcId = link.get('source').id || link.previous('source').id;
```

Comment:
Don't use abbreviations like `src` and `trg`, use `source` and `target` instead.